doc-content/enterprise-only/installation/product-versioning-con.adoc

```diff
@@ -20,13 +20,13 @@ endif::[]
 
 Minor release upgrades::
 
-Minor releases of {PRODUCT} include enhancements, security updates, and bug fixes. Data migration may be required when an application is moved from one minor release to another minor release, such as from {PRODUCT} 7.9.x to 7.10. Automated update tools are often provided with both patch updates and new minor versions of {PRODUCT} to facilitate updating certain components of {PRODUCT}, such as {CENTRAL}, {KIE_SERVER}, and the {HEADLESS_CONTROLLER}. Other {PRODUCT} artifacts, such as the {DECISION_ENGINE} and standalone {CENTRAL}, are released as new artifacts with each minor release and you must re-install them to apply the update.
+Minor releases of {PRODUCT} include enhancements, security updates, and bug fixes. Data migration may be required when an application is moved from one minor release to another minor release, such as from {PRODUCT} 7.10.x to 7.11. Automated update tools are often provided with both patch updates and new minor versions of {PRODUCT} to facilitate updating certain components of {PRODUCT}, such as {CENTRAL}, {KIE_SERVER}, and the {HEADLESS_CONTROLLER}. Other {PRODUCT} artifacts, such as the {DECISION_ENGINE} and standalone {CENTRAL}, are released as new artifacts with each minor release and you must re-install them to apply the update.
```

Comment:
```suggestion
Minor releases of {PRODUCT} include enhancements, security updates, and bug fixes. You may have to migrate your data when an application is moved from one minor release to another minor release, such as from {PRODUCT} 7.10.x to 7.11. {PRODUCT} often provides automated update tools with both patch updates and new minor versions of {PRODUCT} to facilitate updating certain components of {PRODUCT}, such as {CENTRAL}, {KIE_SERVER}, and the {HEADLESS_CONTROLLER}. Red Hat releases other {PRODUCT} artifacts, such as the {DECISION_ENGINE} and standalone {CENTRAL}, as new artifacts with each minor release and you must re-install them to apply the update.
```
Passive voice. Is there an attribute already available for "Red Hat"?

Reply:
I've left it. No attribute.